ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -61,9 +61,14 @@ - (void)applyChangeset:(CKTransactionalComponentDataSourceChangeset *)changeset
                               userInfo:userInfo];
 }
 
-static void applyChangesToCollectionView(CKTransactionalComponentDataSourceAppliedChanges *changes, UICollectionView *collectionView)
+static void applyChangesToCollectionView(CKTransactionalComponentDataSourceAppliedChanges *changes, UICollectionView *collectionView, CKTransactionalComponentDataSourceState *currentState, CKComponentDataSourceAttachController *attachController)
 {
-  [collectionView reloadItemsAtIndexPaths:[changes.updatedIndexPaths allObjects]];
+  NSMutableArray *reloadIndexPaths = @[].mutableCopy;
+  [changes.updatedIndexPaths enumerateObjectsUsingBlock:^(NSIndexPath *indexPath, BOOL *stop) {
+    CKCollectionViewDataSourceCell *cell = (CKCollectionViewDataSourceCell*) [collectionView cellForItemAtIndexPath:indexPath];
+    cell ? _attachToCell(cell, indexPath, currentState, attachController) : [reloadIndexPaths addObject:indexPath];
```

Comment:
If we don't have a cell it only means it's not visible or will be created when we apply changes to the CV, no need to reloadIndexPaths I think.